app/src/main/java/org/mozilla/fenix/components/Services.kt

```diff
@@ -56,25 +44,4 @@ class Services(
             }
         )
     }
-
-    /**
-     * Launches the sign in and pairing custom tab from any screen in the app.
-     * @param context the current Context
-     * @param navController the navController to use for navigation
-     */
-    fun launchPairingSignIn(context: Context, navController: NavController) {
-        // Do not navigate to pairing UI if camera not available or pairing is disabled
-        if (context.hasCamera() && !FeatureFlags.asFeatureFxAPairingDisabled) {
```

Comment:
Ha we were just wondering if we still needed this feature flag the other day. Can you also remove `asFeatureFxAPairingDisabled` from `FeatureFlags`? It's the only place this is used
https://github.com/mozilla-mobile/fenix/issues/9782